### DISCUSSION TITLE
FrskyD not working while others do.

### DISCUSSION THREAD

**Discussion by AlbertFarguell** [pascallanger/DIY-Multiprotocol-TX-Module#756] [Q&A] [answered]
Owning two 4-in-1 Atmel328P modules from Aliexpress, and after uploading the protocols I currently use, everything worked perfectly. These protocols are FrskyD, FrskyX, WFLY2, ASSAN and FlySky.  So, I decided to buy a third unit and after uploading exactly the same software, it turns out that FrskyD doesn't bind, with genuine FrskyD receivers. Given that I force global Id. in all my transmitters I can bind the receiver with another transmitter, but it doesn't work either after binding. Surprisingly enough, it binds and works with FrskyX and ASSAN receivers. Not tested yet with WFLY2 and FlySky.  Any idea about what may be happening?
Thank you in advance,
Albert Farguell

**Comment by pascallanger** [comment, marked as answer]
You need to tune the RF frequency.

**Reply by AlbertFarguell** [reply]
OH! You are faster than light! Thank you very much, I'll try.

**Comment by AlbertFarguell** [comment]
Yes, that was the problem. Thank you again.
